org.envirocar.app/src/org/envirocar/app/views/login/SignupActivity.java

```diff
@@ -363,10 +363,10 @@ private boolean checkUsernameValidity(String username) {
      */
     private boolean checkEmailValidity(String email) {
         boolean isValidEmail = true;
-        if (TextUtils.isEmpty(email)) {
+        if (TextUtils.isEmpty(email.trim())) {
             emailEditText.setError(getString(R.string.error_field_required));
             isValidEmail = false;
-        } else if (!Pattern.matches(EMAIL_REGEX, email)) {
+        } else if (!Pattern.matches(EMAIL_REGEX, email.trim())) {
```

Comment:
Same here

Reply:
Done. I have update the PR with the changes.